src/components/index.js

```diff
@@ -17,3 +17,6 @@ export {
   PrimaryLinkButton,
   ChatbotLoadingButton,
 } from './Button';
+export { DateSelector } from './DateSelector';
+export { Select } from './Select';
+export { TextField } from './TextField';
```

Comment:
why are we not using exports default?
are we going to have more component inside those files? 
feels like they are already big enough.